modules/dataquery/jsx/react.paginator.js

```diff
@@ -1,69 +1,73 @@
-//var PureRenderMixin = require('react/addons').addons.PureRenderMixin;
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
 
-var PaginationLinks = React.createClass({
-    mixins: [React.addons.PureRenderMixin],
-    propTypes: {
-        onChangePage: React.PropTypes.func
-    },
-    getDefaultProps: function() {
-        return {
-            'RowsPerPage' : 10,
-            'Active' : 1
-        }
-    },
-    changePage: function(i) {
-        var that = this;
-        return function(evt) {
-            // Don't jump to the top of the page
-            evt.preventDefault();
+class PaginationLinks extends Component {
```

Comment:
is this component the exact same as what is in `jsx/PaginationLinks.js`?

Reply:
@zaliqarosli I think it's custom for the DQT